utils/get_harbor.sh

```diff
@@ -2,15 +2,21 @@
 
 CURDIR=$PWD
 HARBOR_DIR=".harbor"
+HARBOR_VERSION=${HARBOR_VERSION:-v2.2.3}  # we use this for now
+
+if [[ "${1:-""}" == "" ]]; then
```

Comment:
this can just be `¨$1" == ""` or even the whole if:
```
HARBOR_HOSTNAME="${1?No harbor hostname passed}"
```

Reply:
`¨$1" == ""` would fail because `set -u`, but the other option would probably work!